ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-layering.yaml

```diff
@@ -17,9 +21,14 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
   to: machine-config-operator
+- dockerfile_literal: FROM scratch
+  from: rhel-coreos-base
+  to: rhel-coreos
 promotion:
   disabled: true
   name: layering
+  excluded_images:
```

Comment:
If the above `images` entry is not needed, this shouldn't be needed either since we're not producing an image from the `rhel-coreos` image.